instrumentation/opentelemetry-instrumentation-wsgi/tests/test_wsgi_middleware.py

```diff
@@ -230,6 +241,30 @@ def test_wsgi_internal_error(self):
             StatusCode.ERROR,
         )
 
+    def test_wsgi_metrics(self):
+        app = otel_wsgi.OpenTelemetryMiddleware(error_wsgi_unhandled)
+        self.assertRaises(ValueError, app, self.environ, self.start_response)
+        self.assertRaises(ValueError, app, self.environ, self.start_response)
+        self.assertRaises(ValueError, app, self.environ, self.start_response)
+        metrics_list = self.memory_metrics_reader.get_metrics_data()
+
+        self.assertTrue(len(metrics_list.resource_metrics) != 0)
+        for resource_metric in metrics_list.resource_metrics:
+            self.assertTrue(len(resource_metric.scope_metrics) != 0)
+            for scope_metric in resource_metric.scope_metrics:
+                self.assertTrue(len(scope_metric.metrics) != 0)
+                for metric in scope_metric.metrics:
+                    self.assertIn(metric.name, _expected_metric_names)
+                    data_points = list(metric.data.data_points)
+                    self.assertEqual(len(data_points), 1)
+                    for point in data_points:
+                        if isinstance(point, HistogramDataPoint):
```

Comment:
What about the Gauge point for active requests?

Reply:
I wanted to ask what's the idea/purpose of active requests metrics since its aggregated value turns out to be zero most of times.

Reply:
I think if server is relatively constant load, it should be nonzero. But ya, I had a discussion with a coworker about this as well and he had some ideas to improve things, I think we should raise it to the spec at some point.

I was mostly just asking for purposes of testing that the Gauge is emitted at all

Reply:
Added the test.